sites/offshore-mag/server/templates/website-section/resources-maps-posters.marko

```diff
@@ -0,0 +1,49 @@
+import { getAsObject, getAsArray, get } from '@base-cms/object-path';
+import getAdUnit from '@endeavorb2b/base-website-common/utils/gam/get-adunit';
+import hierarchyAliases from '@endeavorb2b/base-website-common/utils/website-section/hierarchy-aliases';
+
+$ const { site } = out.global;
+$ const section = getAsObject(data, 'section');
+$ const aliases = hierarchyAliases(section);
+$ const adSlots = {
+  'gpt-ad-lb1': getAdUnit(site, { name: 'lb1', aliases }),
+  'gpt-ad-rail1': getAdUnit(site, { name: 'rail1', aliases, size: [300, 250] }),
+  'gpt-ad-rail2': getAdUnit(site, { name: 'rail2', aliases, size: [300, 600] }),
```

Comment:
Additionally, if you don't need to include multiple units for a single request, just use the alternate form -- don't include the getAdUnit calls, don't send `slots` to `endeavor-ad-gam-head`, and call `endeavor-gam-ad-unit-define-display name="lb1"` below instead of `-display`.

Reply:
@solocommand  Can you take a look again and see if that is better?